__tests__/news.test.js

```diff
@@ -176,3 +176,44 @@ describe("GET/api/articles", () => {
       });
   });
 });
+
+describe("GET/api/articles/:article_id/comments", () => {
+  test("responds with array of comments with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.comments).toBeInstanceOf(Array);
+        result.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+          });
+        });
+      });
+  });
+  test("responds with correct error message", () => {
```

Comment:
Best to leave incomplete tests out of pull requests - either comment these out or lose them altogether until you need them

Reply:
I was under the impression this wasnt incomplete as im just testing for correct error codes